src/plugins/vis_type_timeseries/public/application/visualizations/views/timeseries/index.js

```diff
@@ -56,7 +56,6 @@ const handleCursorUpdate = (cursor) => {
 };
 
 export const TimeSeries = ({
-  darkMode,
```

Comment:
We should also remove the darkMode prop Type on L276

Reply:
I removed the `darkMode` as it was only used to pass to the `getTheme` function to select the `baseTheme` from `@elastic/charts` which is what the `useChartBaseTheme` hook does on it's own.